src/player.js

```diff
@@ -23,8 +23,8 @@ class Player {
      */
     constructor(element, options = {}) {
         /* global jQuery */
-        if ('jQuery' in window && element instanceof jQuery) {
```

Comment:
I think the simpler change here is to also check that jQuery is not `undefined`:

```js
if ('jQuery' in window && typeof window.jQuery !== 'undefined' && element instanceof jQuery) {
```